regimes/au/tax_categories.go

```diff
@@ -0,0 +1,111 @@
+package au
+
+import (
+	"time"
+
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+var taxCategories = []*tax.CategoryDef{
+	{
+		Code: tax.CategoryGST,
+		Name: i18n.String{
+			i18n.EN: "GST",
+		},
+		Title: i18n.String{
+			i18n.EN: "Goods and Services Tax",
+		},
+		Retained: false,
+		Rates: []*tax.RateDef{
+			{
+				Key: tax.RateExempt,
+				Name: i18n.String{
+					i18n.EN: "GST-free",
+				},
+				Values: []*tax.RateValueDef{
+					{
+						Since:   cal.NewDate(2000, time.July, 1),
+						Percent: num.MakePercentage(0, 2),
+					},
+				},
+			},
+			{
+				Key: tax.RateStandard,
+				Name: i18n.String{
+					i18n.EN: "Standard Rate",
+				},
+				Values: []*tax.RateValueDef{
+					{
+						Since:   cal.NewDate(2000, time.July, 1),
+						Percent: num.MakePercentage(10, 2),
+					},
+				},
+			},
+			{
+				Key: tax.RateZero,
+				Name: i18n.String{
+					i18n.EN: "Input-Taxed",
+				},
+				Values: []*tax.RateValueDef{
+					{
+						Since:   cal.NewDate(2000, time.July, 1),
+						Percent: num.MakePercentage(0, 2),
+					},
+				},
+			},
+		},
+	},
+	//Source: https://www.ato.gov.au/businesses-and-organisations/gst-excise-and-indirect-taxes/wine-equalisation-tax
```

Comment:
This is good, but you can add it to the raw data in the "Source" attribute. The `fr` regime has an example of this.